examples/graphene_apps/python_workloads/setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Please keep this file in $TCF_HOME/tc/graphene/python_worker

Reply:
This setup.py is to build python workloads module. $TCF_HOME/tc/graphene/python_worker module has separate Makefile and setup.py